pppd/tls.c

```diff
@@ -186,12 +186,13 @@ static int tls_verify_callback(int ok, X509_STORE_CTX *ctx)
 
         /* Match the suffix of common name */
         if (!strcmp(TLS_VERIFY_SUFFIX, tls_verify_method)) {
-            int len = strlen(ptr1);
-            int off = strlen(cn_str) - len;
+            size_t len1, len2;
             ptr2 = cn_str;
-            if (off > 0) {
-                ptr2 = cn_str + off;
-            }
+
```

Comment:
Did you test this change? 

As am scrolling through, reviewing this to ensure the functionality have not changed is a bit hard. Though the change make the code look cleaner, does it still work?

Reply:
Yes, absolutely.